gufe/components/explicitmoleculecomponent.py

```diff
@@ -85,12 +86,15 @@ def _check_partial_charges(mol: RDKitMol) -> None:
         wmsg = f"Partial charges provided all equal to " "zero. These may be ignored by some Protocols."
         warnings.warn(wmsg)
     else:
-        wmsg = (
+        message = (
             "Partial charges have been provided, these will "
             "preferentially be used instead of generating new "
-            "partial charges"
+            "partial charges."
         )
-        warnings.warn(wmsg)
+        if logger is None:
```

Comment:
Probably needs its own "calling the method directly" test.

Reply:
done!